docs/contributing.md

```diff
@@ -0,0 +1,48 @@
+# SSS Contributor Guideline
+
+## First Steps
+
+Some good first steps and best pracices when using Git are explaind here:
+- the Git Immersion tutorial: https://gitimmersion.com/
+- the Atlassian tutorial: https://www.atlassian.com/git/tutorials/learn-git-with-bitbucket-cloud
+- this blog post on the ROSEdu Techblog: https://techblog.rosedu.org/git-good-practices.html
+
+## Content Writing Style
+
+This section addresses the development of session content and other Markdown files.
+
+Write each sentence on a new line.
+This way, changing one sentence only affects one line in the source code. 
+
+Use the **first person plural** when writing documentation and tutorials.
+Use phrases like "we run the command / app", "we look at the source code", "we find the flag".
+
+Use the second person for challenges and other individual activities.
+Use phrases like "find the flag", "run this command", "download the tool".
+
+## Issues
+
+When opening an issue, please clearly state the problem.
+Make sure it's reproducible.
+Add images if required.
+Also, if relevant, detail the environment you used (OS, software versions).
+Ideally, if the issue is something you could fix, open a pull request with the fix.
+
+## Discussions
+
+Use discussions for bringing up ideas on content, new chapters, new sections.
```

Comment:
```suggestion
Use GitHub discussions for bringing up ideas on content, new chapters, new sections.
```

Reply:
"For in-depth discussions, please join the [SSS Discord community](https://discord.gg/qx3RmNEVkB)."